persistence-modules/pom.xml

```diff
@@ -24,6 +24,7 @@
         <module>hibernate-mapping</module> <!-- long running -->
         <module>hibernate-ogm</module>
         <module>hibernate-annotations</module>
+        <module>hibernate-types</module>
```

Comment:
Is this ok, or is this change made after the article is published?

Reply:
In the previous PR you said to remove this because it is usually done after the article is published. But the author docs say to add it. Which one should I do?

Reply:
in the previous PR I said I didn't understand what you meant, I still don't :)

Reply:
@rpvilao Section _"how to create a new module"_ in _"baeldung - for authors - 4 - implementation code"_ says "if the new module is a sub-module, then this should be added to the pom of their parent, instead of the main pom", but you told me not to.

Which one is it.

Reply:
I just looked back at the old PR. You didn't comment on this. I got it mixed up with the readme file.